### ISSUE TITLE
The reload prompt doesn't default to positive answer (even though the capital 'Y' suggests it)

### ISSUE THREAD

**Issue by mik13ST** [karli-sjoberg/gswitch#7]
While getting this (potentially) awesome tool to work on my setup I was confused with `Unknown argument: .` error. So I started inspecting the source code and discovered the reason. The prompt does not default to the positive choice despite the 'Y' being a capital. This convention is present in many UNIX tools (usually package managers). What was your reason to make the 'Y' a capital?

**Actual behavior (confusing error):**
```
$ sudo bash ./gswitch internal
You are already set up. Would you like to reload? (Y/n)

Unknown argument: .
```

**Expected behavior (tool attempts to do the switch - ignore the failures):**
```
$ sudo bash ./gswitch internal
You are already set up. Would you like to reload? (Y/n)

Failed to stop display-manager.service: Unit display-manager.service not loaded.
Unit display-manager.service could not be found.
modprobe: FATAL: Module nvidia_drm is in use.
modprobe: FATAL: Module nvidia_modeset is in use.
modprobe: FATAL: Module nvidia is in use.
Failed to start display-manager.service: Unit display-manager.service not found.
```

TLDR: I think pressing the Enter key should be recognized as positive answer as well.

Should I attempt to fix it myself and submit a pull request?

**Comment by karli-sjoberg** [comment]
First of all, thank you for using the software and Merry Christmas!

The choice is actually on purpose default-less because answering "Y|y|Yes|yes" will _forcibly_ close your current window session, killing any potential unsaved documents etc. you might have. Therefore, having Enter as a default might make it too easy executing this by accident.

Are you just disturbed by the "Y" being capital and therefore not following some common convention, we can change that instead! Although, in my experience, if a default is used, it's usually marked between brackets:
(Y/n) [Yes]

Which it isn't, so no default should be expected. Thoughts?

**Comment by mik13ST** [comment]
Or right, it's Christmas. Merry Christmas, I guess.

That makes sense. I haven't actually experienced a successful switch of graphics card this tool is used for. In that case not having an easy to trigger default option is a good idea.

Yes, that is the root of the problem I have with the prompt. Take a look at `apt` or `pacman`. They both have a confirmation prompt with a default choice written in capitals (or just the first letter in capitals). Then look at `ssh`. If you are connecting to an untrusted server the prompt does not have a default value (Enter key does nothing) and none of the options are capitalized. Based on this, I think this tool should not capitalize the 'Y' letter (or capitalize both). I haven't seen the brackets notation much.

The simplest solution is changing the prompt to `(y/n)` or `(yes/no)` (`ssh` uses this, I think). Since this is a tiny change and kind of a nitpick, I could actually use this to learn to use the GitHub work flow (forking, committing and submitting a pull request). I have never contributed to an open source project like this.

**Comment by karli-sjoberg** [comment]
Sure man, sounds good! I'll be awaiting your pull request then 😊

**Comment by karli-sjoberg** [comment]
I've thought long and hard on this issue and finally decided against this change. The reason to still have a capital 'Y' is to show to the user that it's ok to write either 'Yes', 'YES', 'yes', 'No', 'NO' or 'no', it doesn't matter. Your example about package managers, while true for some, it's different in other tools, like 'zypper' e.g. There, just like I said, you have the default input in between brackets.
